lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/classloader/ClassLoaderSafeEventListener.java

```diff
@@ -27,6 +29,7 @@
     private final EventListener delegate;
     private final ClassLoader classLoader;
 
+    @Inject
     public ClassLoaderSafeEventListener(@ForClassLoaderSafe EventListener delegate, ClassLoader classLoader)
```

Comment:
So `@ForClassLoaderSafe` was not working yet. Do we have other such places?

Reply:
Yes..Need to check for others